dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/user/RoleAssignmentRepository.java

```diff
@@ -23,6 +22,12 @@ public RoleAssignmentRepository() {
                 .getResultList();
     }
 
+    public List<RoleAssignment> findByDefinitionPointId(List<Long> definitionPointIds) {
```

Comment:
Maybe we could rename this to `findByDefinitionPointIds` - to distinguish version that operates on single id?